src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -2142,7 +2142,6 @@ private static unsafe nuint NarrowUtf16ToAscii_Intrinsified_256(char* pUtf16Buff
             // jumps as much as possible in the optimistic case of "all ASCII". If we see non-ASCII
             // data, we jump out of the hot paths to targets at the end of the method.
 
-            Debug.Assert(Vector256.IsHardwareAccelerated, "Vector256 is required.");
```

Comment:
There are similar asserts in multiple other places, e.g. here: https://github.com/dotnet/runtime/blob/bd0d278b5ccfbd7a6b50f6d462d838d6eaeaa00f/src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs#L2261 . Do all of them have the same problem?

Reply:
I'm curious about this as well.

Reply:
I think, .NET Runtime currently have some bug inside [morph.cpp](https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/morph.cpp). May be it's the same as previous which was fixed some years before by https://github.com/dotnet/coreclr/pull/15814
Could anybody from .NET team check my guess?

Reply:
Could anybody view changes inside my branch [bugfix/trying-to-fix-bug-with-build](https://github.com/Maximys/runtime/commits/bugfix/trying-to-fix-bug-with-build)? I think it fix problem with current build (currently I check it). If so, I can create PR or somebody from .NET team can cherry pick [my commit](https://github.com/Maximys/runtime/commit/d474d674e571a9ef5f652c91463b0354809e5ad6)
FYI @ivdiazsa

Reply:
@Maximys - I believe we intentionally removed the -O Crossgen2 option as it should be implied by assembly-level attributes set by the Roslyn compiler. Can you please be more specific as to what exactly is failing for you?

Reply:
@trylek , I have next result of _build.cmd_ run without _-O Crossgen2_:
```
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe
rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_
STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE
TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro
soft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ilationThread|3(Object objThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 811 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.
NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Threading.Thread.StartCallback() [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsof
t.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe
rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_
STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE
TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro
soft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileMethodList|2(IEnumerable`1 methodList) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 793 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micr
osoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.ComputeDependencyNodeDependencies(List`1 obj) in C:\runtime\src\
coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 717 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeDependencies(List`1 deferredStaticDep
endencies) in C:\runtime\src\coreclr\tools\aot\ILCompiler.DependencyAnalysisFramework\DependencyAnalyzer.cs:line 176 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj
]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeMarkedNodes() in C:\runtime\src\corec
lr\tools\aot\ILCompiler.DependencyAnalysisFramework\DependencyAnalyzer.cs:line 316 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.Compile(String outputFile) in C:\runtime\src\coreclr\tools\aot\I
LCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 387 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.RunSingleCompilation(Dictionary`2 inFilePaths, InstructionSetSupport instructionSetSu
pport, String compositeRootPath, Dictionary`2 unrootedInputFilePaths, HashSet`1 versionBubbleModulesHash, ReadyToRunCompilerContext typeSystemContext, Logger logger) in C:\runtime\src\coreclr\tools\aot\crossgen
2\Program.cs:line 637 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.Run() in C:\runtime\src\coreclr\tools\aot\crossgen2\Program.cs:line 302 [C:\runtime\s
rc\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Crossgen2RootCommand.<>c__DisplayClass205_0.<.ctor>b__0(ParseResult result) in C:\runtime\src
\coreclr\tools\aot\crossgen2\Crossgen2RootCommand.cs:line 261 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult parseResult) [C:\runtime\src\install
er\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.Main(String[] args) in C:\runtime\src\coreclr\tools\aot\crossgen2\Program.cs:line 914
 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printMethodName(CORINFO_METHOD_STRUCT_* ftn, Byte* buffer, UIntPtr buf
ferSize, UIntPtr* requiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 3279 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printMethodName(IntPtr thisHandle, IntPtr* ppException, CORINFO_METHO
D_STRUCT_* ftn, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 1690 [C:\runtime\src\installer\pkg\sfx\Microsoft
.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro
soft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ilationThread|3(Object objThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 811 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.
NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Threading.Thread.StartCallback() [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsof
t.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe
rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_
STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE
TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro
soft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileMethodList|2(IEnumerable`1 methodList) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 793 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micr
osoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.ComputeDependencyNodeDependencies(List`1 obj) in C:\runtime\src\
coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 717 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeDependencies(List`1 deferredStaticDep
endencies) in C:\runtime\src\coreclr\tools\aot\ILCompiler.DependencyAnalysisFramework\DependencyAnalyzer.cs:line 176 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj
]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.DependencyAnalysisFramework.DependencyAnalyzer`2.ComputeMarkedNodes() in C:\runtime\src\corec
lr\tools\aot\ILCompiler.DependencyAnalysisFramework\DependencyAnalyzer.cs:line 316 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.Compile(String outputFile) in C:\runtime\src\coreclr\tools\aot\I
LCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 387 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.RunSingleCompilation(Dictionary`2 inFilePaths, InstructionSetSupport instructionSetSu
pport, String compositeRootPath, Dictionary`2 unrootedInputFilePaths, HashSet`1 versionBubbleModulesHash, ReadyToRunCompilerContext typeSystemContext, Logger logger) in C:\runtime\src\coreclr\tools\aot\crossgen
2\Program.cs:line 637 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.Run() in C:\runtime\src\coreclr\tools\aot\crossgen2\Program.cs:line 302 [C:\runtime\s
rc\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Crossgen2RootCommand.<>c__DisplayClass205_0.<.ctor>b__0(ParseResult result) in C:\runtime\src
\coreclr\tools\aot\crossgen2\Crossgen2RootCommand.cs:line 261 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.CommandLine.Invocation.InvocationPipeline.Invoke(ParseResult parseResult) [C:\runtime\src\install
er\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.Program.Main(String[] args) in C:\runtime\src\coreclr\tools\aot\crossgen2\Program.cs:line 914
 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe
rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_
STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE
TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro
soft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ilationThread|3(Object objThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 811 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.
NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Threading.Thread.StartCallback() [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsof
t.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor
e.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s
fxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r
untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff
er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b
ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,
 UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe
rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_
STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE
TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int
Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL
methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge
r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp
ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s
fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]
C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Mi8:36soft.NETCore.App.Runtime.sfxproj]                                                                                                                                                                                  C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp ilationThread|3(Object objThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 811 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft. NETCore.App.Runtime.sfxproj]                                                                                                                                                                                       C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Threading.Thread.StartCallback() [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsof t.NETCore.App.Runtime.sfxproj]                                                                                                                                                                                     C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Process terminated. Assertion failed. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCor e.App.Runtime.sfxproj]                                                                                                                                                                                             C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error : Vector256 is required. [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.s fxproj]                                                                                                                                                                                                            C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Ascii.NarrowUtf16ToAscii(Char* pUtf16Buffer, Byte* pAsciiBuffer, UIntPtr elementCount) [C:\r untime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                                                                                                                          C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8Utility.TranscodeToUtf8(Char* pInputBuffer, Int32 inputLength, Byte* pOutputBuff er, Int32 outputBytesRemaining, Char*& pInputBufferRemaining, Byte*& pOutputBufferRemaining) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                        C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Text.Unicode.Utf8.FromUtf16(ReadOnlySpan`1 source, Span`1 destination, Int32& charsRead, Int32& b ytesWritten, Boolean replaceInvalidSequences, Boolean isFinalBlock) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                                                 C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.PrintFromUtf16(ReadOnlySpan`1 utf16, Byte* buffer, UIntPtr bufferSize,  UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1911 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]         C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.printClassName(CORINFO_CLASS_STRUCT_* cls, Byte* buffer, UIntPtr buffe rSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 1975 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]   C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl._printClassName(IntPtr thisHandle, IntPtr* ppException, CORINFO_CLASS_ STRUCT_* cls, Byte* buffer, UIntPtr bufferSize, UIntPtr* pRequiredBufferSize) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl_generated.cs:line 547 [C:\runtime\src\installer\pkg\sfx\Microsoft.NE TCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                                                                                                                                                                   C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                      C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.JitCompileMethod(IntPtr& exception, IntPtr jit, IntPtr thisHandle, Int Ptr callbacks, CORINFO_METHOD_INFO& info, UInt32 flags, IntPtr& nativeEntry, UInt32& codeSize) [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                      C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL  methodIL) in C:\runtime\src\coreclr\tools\Common\JitInterface\CorInfoImpl.cs:line 352 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                               C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at Internal.JitInterface.CorInfoImpl.CompileMethod(MethodWithGCInfo methodCodeNodeNeedingCode, Logger logge r) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\JitInterface\CorInfoImpl.ReadyToRun.cs:line 801 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]        C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp ileOneMethod|5(DependencyNodeCore`1 dependency, Int32 compileThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 899 [C:\runtime\src\installer\pkg\s fx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj]                                                                                                                                                    C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp ileOnThread|4(Int32 compilationThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 833 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Micro soft.NETCore.App.Runtime.sfxproj]                                                                                                                                                                                  C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at ILCompiler.ReadyToRunCodegenCompilation.<>c__DisplayClass50_0.<ComputeDependencyNodeDependencies>g__Comp ilationThread|3(Object objThreadId) in C:\runtime\src\coreclr\tools\aot\ILCompiler.ReadyToRun\Compiler\ReadyToRunCodegenCompilation.cs:line 811 [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsoft. NETCore.App.Runtime.sfxproj]                                                                                                                                                                                       C:\runtime\artifacts\bin\Crossgen2Tasks\Debug\net8.0\Microsoft.NET.CrossGen.targets(469,5): error :    at System.Threading.Thread.StartCallback() [C:\runtime\src\installer\pkg\sfx\Microsoft.NETCore.App\Microsof t.NETCore.App.Runtime.sfxproj]
```